cosmwasm/packages/wasmi-runtime/src/crypto/aes_siv.rs

```diff
@@ -1,3 +1,4 @@
+use crate::crypto::errors::CryptoError;
```

Comment:
should be under the description

Reply:
true, my bad

Reply:
done.
Oddly, rustfmt insisted on putting it above the comment. I have no idea why O.o